test/create.test.js

```diff
@@ -112,3 +111,85 @@ test.skip('tribes.create (opts.addPOBox)', (t) => {
     t.end()
   })
 })
+
+test('create reuses an unused group feed (because of an earlier crash or something)', (t) => {
+  const server = Testbot()
+
+  function countGroupFeeds(cb) {
+    pull(
+      server.metafeeds.branchStream({ live: false }),
```

Comment:
branchStream currently isn't outputting anything, even when i know i have at least a root and group feed

Reply:
The default value for `old` is false https://github.com/ssbc/ssb-meta-feeds/blob/58454e8440eab0bd64ed0bf63c6cd9a9c8aaf5d1/lookup.js#L306-L309 so in this case you have both live and old false, which means it'll return pull.empty() https://github.com/ssbc/ssb-meta-feeds/blob/58454e8440eab0bd64ed0bf63c6cd9a9c8aaf5d1/lookup.js#L336

Reply:
One way to simulate a crash with a partial state is to manually create an empty group feed in the test code, instead of using ssb.tribes2.create.

Reply:
> The default value for old is false

wow lmao it literally returns empty every time

> manually create an empty group feed in the test code

yep that's what i ended up doing

Reply:
> wow lmao it literally returns empty every time

Perhaps it could return a `pull.error()` stream instead.